resources/zib/ext-NursingIntervention-ContributorIsRequester.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-NursingIntervention-ContributorIsRequester" />

```

Comment:
ext-NursingIntervention.ContributorIsRequester following profiling guidelines

.url, .name and .title follow

Reply:
Eh ... what do you mean?

Reply:
Should be a dot between NursingIntervention and ContributorIsRequester imho.

> This extension doesn't represent a specific concept from the zib.

I think it does implicitly.

Reply:
No, "ContributorIsRequester" is not a concept from the zib, "Requester" is. This extension doesn't carry the information in Requester, it marks something already present as "this is the narrower concept from the zib". When you'd add the zib concept to an existing resource, the zib.concept name ("NursingIntevention.Requester") would make sense; but this is not the case here.

Reply:
I understand, but that is not really my point. I would still expect either 'CarePlan-ContributorIsRequester' or 'NursingIntervention.ContributorIsRequester' here. You can argue that the extension is reusable for the resource or that it represents a concept specifically for the zib (although this would be a _new_ concept, it is still specific for the zib and necessary for a correct interpretation of the concept 'Requester')